app/sign-in/[[...sign-in]]/sign-in-client.tsx

```
@@ -0,0 +1,53 @@
+  }
+
+  // If already signed in, show a message
+  if (isSignedIn) {
+    return (
+      <div className="flex min-h-screen items-center justify-center py-12 flex-col">
+        <div className="mb-4 text-green-600">
+          <svg xmlns="http://www.w3.org/2000/svg" className="h-12 w-12" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+            <path strokeLinecap="round" strokeLinejoin="round" strokeWidth={2} d="M5 13l4 4L19 7" />
+          </svg>
+        </div>
+        <h2 className="text-2xl font-bold mb-2">You're already signed in!</h2>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)